config/locales/api.en.yml

```diff
@@ -1217,6 +1216,64 @@ en:
             }
           }
 
+    design_factor:
+      attrs:
+        -
+          name: id
+        -
+          name: design_factor_name
+          create: required
+          desc: An optional name assigned to this design factor
+        -
+          name: institute_id
+          create: required
+          desc: Name of the institute in case which designed the experiment
+        -
+          name: trial_location_name
+          create: required
+          desc: Optional name of the location where this design was used
+        -
+          name: design_unit_counter
+          create: required
+          desc: The ordinal number of a unique design factor for a given plant trial; it is safe to set it to the number of the smallest design factors (e.g. "354")
+        -
+          name: comments
+        -
+          name: entered_by_whom
+        -
+          name: date_entered
+        -
+          name: data_owned_by
+        -
+          name: data_provenance
+        -
+          name: confirmed_by_whom
+        -
+          name: design_factors
+          format: array
```

Comment:
Before, we denoted arrays as "int[]" (only integer arrays were in use). So, I would propose to either stick to this convention, or maybe change "int[]" to "array" too?